lib/approvals/writer.rb

```diff
@@ -16,16 +16,13 @@ module Writer
       html: Writers::HtmlWriter.new,
       hash: Writers::HashWriter.new,
       array: Writers::ArrayWriter.new,
+      txt: Writers::TextWriter.new,
     }
 
 
     class << self
       def for(format)
-        if REGISTRY.include?(format)
-          REGISTRY[format]
-        else
-          TextWriter.new
-        end
+        REGISTRY[format] || format.new
       end
```

Comment:
My first thought was that it's fine if it blows up, as long as it's easy to see why. On the other hand, it might be nice to tell them what to do to fix it (pass in a supported writer, I guess?)